cmake/AddChmTarget.cmake

```diff
@@ -20,7 +20,7 @@ function (add_chm_target docname lang entities figures)
            "-Dentities=\"${entities}\""
            -D HHC=${HHC}
            -P ${CMAKE_SOURCE_DIR}/cmake/MakeChm.cmake
-        DEPENDS ${entities} "${docname}.xml" "${CMAKE_SOURCE_DIR}/docbook/gnc-docbookx.dtd" ${figures}
+        DEPENDS ${entities} "index.docbook" "${CMAKE_SOURCE_DIR}/docbook/gnc-docbookx.dtd" ${figures}
```

Comment:
As we finally have to use "index.docbook" IMHO we should rename in our sources *.xml to *.docbook.